### PR TITLE
schismtracker: update to 20240909

### PR DESCRIPTION
**Pull request by knightcole**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)

### CODE REVIEW

File: srcpkgs/schismtracker/template [hunk 1]
```diff
@@ -1,16 +1,16 @@
 # Template file for 'schismtracker'
 pkgname=schismtracker
-version=20240809
+version=20240909
 revision=1
 build_style=gnu-configure
-hostmakedepends="pkg-config autoconf-archive automake SDL2-devel python3 git libltdl-devel libflac-devel"
+hostmakedepends="pkg-config autoconf-archive automake SDL2-devel python3 git libltdl-devel libflac-devel libutf8proc-devel"
```
Comment (abenson) [inline srcpkgs/schismtracker/template:6]: This goes `makedepends`

Comment (abenson) [inline srcpkgs/schismtracker/template:6]:
```suggestion
hostmakedepends="pkg-config autoconf-archive automake SDL2-devel python3 git"
```

File: srcpkgs/schismtracker/template [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
 makedepends="SDL2-devel"
```
Comment (abenson) [inline srcpkgs/schismtracker/template:7]:
```suggestion
makedepends="SDL2-devel libutf8proc-devel"
```

Comment (knightcole) [reply srcpkgs/schismtracker/template]: Thanks a lot!

**Pull request merged**